### PR TITLE
app-text/pandoc: Add metapackage to ease transition to pandoc-cli

### PR DESCRIPTION
**Pull request by ulm**

### CODE REVIEW

**Comment by ulm** [comment]
@hololeap

File: app-text/pandoc/pandoc-3.ebuild [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+EAPI=8
+
+DESCRIPTION="Metapackage for pandoc version 3"
+HOMEPAGE="https://pandoc.org/"
+
+LICENSE="metapackage"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~ppc64 ~riscv ~x86"
+
+RDEPEND="app-text/pandoc-cli"
```
Comment (hololeap) [inline app-text/pandoc/pandoc-3.ebuild:13]: Should there be an `elog` message in `pkg_postinst` letting the users know that this is now a metapackage and that the real package is `app-text/pandoc-cli`?

Comment (ulm) [reply app-text/pandoc/pandoc-3.ebuild:13]: Any suggestion for a wording?

Comment (hololeap) [reply app-text/pandoc/pandoc-3.ebuild:13]:
```ebuild
elog "The pandoc CLI executable has been split off upstream into a new package named"
elog "pandoc-cli, starting with pandoc-3. This metapackage ${P}"
elog "was created to help ease upgrading, but it is recommended to switch to"
elog "app-text/pandoc-cli."
```

**Pull request closed without merging**